pyscriptjs/tests/test_altair.py

```diff
@@ -0,0 +1,86 @@
+"""Test: altair"""
+
+import math
+import re
+import time
+
+try:
+    # When running `make test`
+    from .common import MAX_TEST_TIME, TEST_TIME_INCREMENT, _url_join
+except ImportError:
+    # When running `python test_<example name>.py`
+    from common import (
+        MAX_TEST_TIME,
+        TEST_TIME_INCREMENT,
+        _url_join,
+    )
+
+from playwright.sync_api import sync_playwright
+
+BASE_URL = "http://127.0.0.1:8080"
+EXPECTED_TITLE = "Altair"
+HTML_FILE = "altair.html"
+
+
+def test():
```

Comment:
Can you rename the test function here? test functions should have proper names to increase semantics and allow for better test management, so something like `test_altair` should be just fine